exercises/01-getting-started.md

```diff
@@ -20,6 +20,23 @@ On your laptop open a command line and enter the following command. Don't forget
 git clone https://github.com/<your-github-username>/cloud-sdk-team-calendar.git
 ```
 
+## Deploy the S/4HANA and SFSF mock server (Optional)
+
+>Note: If you have access to both SAP S/4HANA and SAP Successfactors Cloud systems, you can skip this step.
+
+Follow these [instructions](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to setup your S4/HANA / SFSF mock server and deploy it on SAP Cloud Platform, Cloud Foundry.
+
+## Setup the destination file
+
+Retrieve the server's route, you can then replace the S4HANA and SFSF `url` in the [`.env`](../.env) file with the route url of the server. The `.env` file should have the same URLs if you use the mock server, because the mock server offers both services:
```

Comment:
```suggestion
## Setup the destinations environment variable

Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system.
```